acceptance/pre_suite/00_install_agent.rb

```diff
@@ -0,0 +1,20 @@
+require 'beaker-puppet'
```

Comment:
It might be more useful to have one file for the pre-suite, `setup_master.rb`, so that it is apparent that this setup's only for the master? The individual files are small enough that I think splitting them up into steps under a single test would still give the right amount of granularity in case something fails. As things currently are, I have to dig into these files in order to see that they apply only for the master.

Reply:
I've merged these into a single file